pkg/sharing/secret_exports.go

```diff
@@ -0,0 +1,98 @@
+package sharing
+
+import (
+	"sync"
+
+	"github.com/go-logr/logr"
+	sgv1alpha1 "github.com/vmware-tanzu/carvel-secretgen-controller/pkg/apis/secretgen/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+type SecretExports struct {
+	log logr.Logger
+
+	exportedSecretsLock sync.RWMutex
+	exportedSecrets     map[string]exportedSecret
+}
+
+type exportedSecret struct {
+	*sgv1alpha1.SecretExport
+	*corev1.Secret
+}
+
+func (p exportedSecret) Key() string {
```

Comment:
for the receivers here, `p` is a little oPaque since it's a middle letter...  feelings on `es`?  or `eSecret` if you want to sound like a dotcom era company?

Reply:
done.